lightning/src/ln/functional_tests.rs

```diff
@@ -5507,12 +5509,14 @@ fn test_dynamic_spendable_outputs_local_htlc_success_tx() {
 	};
 
 	mine_transaction(&nodes[1], &node_tx);
-	connect_blocks(&nodes[1], ANTI_REORG_DELAY - 1);
+	connect_blocks(&nodes[1], BREAKDOWN_TIMEOUT as u32 - 1);
```

Comment:
Probably outside the scope of the PR, but I feel like `BREAKDOWN_TIMEOUT` could use a better name.

Reply:
Probably, the fact that its public with such an awkward name kinda sucks.